src/index.js

```diff
@@ -33,7 +33,7 @@ export default class Shevy {
     this.baseSpacing = this.baseSpacing.bind(this)
 
     // Set headings
-    baseFontScale.forEach((factor, index) => {
+    this.baseFontScale.forEach((factor, index) => {
```

Comment:
Good catch.

Reply:
Test for the win here. Failed one and was able to fine this.